kratos/geometries/quadrature_point_geometry.h

```diff
@@ -452,7 +452,7 @@ class QuadraturePointGeometry
 
     // quatrature point can be related to a parent geometry. To keep the connection,
     // this geometry is related to the integration point.
-    GeometryType* mpGeometryParent;
+    GeometryType* mpGeometryParent = nullptr;
```

Comment:
out of curiosity, was this a problem before?
I thought it would be the same

Reply:
My understanding is that `nullptr` is roughtly `0x0` and unitialized is like uninitalized variables, probably `0x0`, but not guaranteed, so code hanging from an eventual `if (mpGeometryParent != nullptr)` is dangerous at best

Reply:
Carlo got it in one 😄, I  added this to check if the geom has a parent or not (without having to add a Has function to the core 😱).  Is there a different but similarly simple way to do a safe check of this?